diff --git a/plugins/optim/wscript b/plugins/optim/wscript
index c755303c..da78c906 100644
--- a/plugins/optim/wscript
+++ b/plugins/optim/wscript
@@ -50,10 +50,14 @@ def configure (conf):
 		if clp_dir:
 			clp_include = os.path.join (clp_dir, "include")
 			# Cpl install everything in coin directory
-			coin_include = os.path.join (clp_include, "coin")
+			coin_include1 = os.path.join (clp_include, "coin")
+			coin_include2 = os.path.join (clp_include, "clp/coin")  # for OSX
+			coin_include3 = os.path.join (clp_include, "coinutils/coin")  # for OSX
 			clp_lib = os.path.join (clp_dir, "lib")
 			conf.env.append_unique ("INCLUDES_IBEX_DEPS", clp_include)
-			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include)
+			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include1)
+			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include2)
+			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include3)
 			conf.env.append_unique ("LIBPATH_IBEX_DEPS", clp_lib)
 		else:
 			clp_include = ""
