diff --git a/plugins/optim/wscript b/plugins/optim/wscript
index da78c906..37093048 100644
--- a/plugins/optim/wscript
+++ b/plugins/optim/wscript
@@ -54,7 +54,7 @@ def configure (conf):
 			coin_include2 = os.path.join (clp_include, "clp/coin")  # for OSX
 			coin_include3 = os.path.join (clp_include, "coinutils/coin")  # for OSX
 			clp_lib = os.path.join (clp_dir, "lib")
-			conf.env.append_unique ("INCLUDES_IBEX_DEPS", clp_include)
+			# conf.env.append_unique ("INCLUDES_IBEX_DEPS", clp_include)
 			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include1)
 			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include2)
 			conf.env.append_unique ("INCLUDES_IBEX_DEPS", coin_include3)
diff --git a/wscript b/wscript
index 1e0e83aa..3f6d6b27 100644
--- a/wscript
+++ b/wscript
@@ -7,7 +7,7 @@ from waflib import Scripting, Logs, Options, Utils
 import ibexutils
 
 # The following variable is used to build ibex.pc and by "waf dist"
-VERSION="2.3.0"
+VERSION="2.5.1"
 # The following variable is used only by "waf dist"
 APPNAME='ibex-lib'
 
