diff --git a/wscript b/wscript
index 1e0e83aa..9f6f551d 100644
--- a/wscript
+++ b/wscript
@@ -7,7 +7,7 @@ from waflib import Scripting, Logs, Options, Utils
 import ibexutils
 
 # The following variable is used to build ibex.pc and by "waf dist"
-VERSION="2.3.0"
+VERSION="2.5.1"
 # The following variable is used only by "waf dist"
 APPNAME='ibex-lib'
 
@@ -82,8 +82,8 @@ def configure (conf):
 	conf.env.append_unique ("LIBPATH_IBEX_DEPS", conf.env.LIBDIR)
 	conf.env.INCDIR = os.path.join (conf.env.PREFIX, "include")
 	conf.env.INCDIR_HDR = os.path.join (conf.env.INCDIR, "ibex")
-	conf.env.append_unique ("INCLUDES_IBEX_DEPS", conf.env.INCDIR)
-	conf.env.append_unique ("INCLUDES_IBEX_DEPS", conf.env.INCDIR_HDR)
+	# conf.env.append_unique ("INCLUDES_IBEX_DEPS", conf.env.INCDIR)
+	# conf.env.append_unique ("INCLUDES_IBEX_DEPS", conf.env.INCDIR_HDR)
 	conf.env.INCDIR_3RD = os.path.join (conf.env.INCDIR_HDR, "3rd")
 	conf.env.LIBDIR_3RD = os.path.join (conf.env.LIBDIR, "ibex", "3rd")
 	conf.env.PKGDIR = os.path.join (conf.env.PREFIX, "share", "pkgconfig")
